lua/lspconfig/volarvuels.lua

```diff
@@ -0,0 +1,88 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local server_name = 'volarvuels'
+
+-- TODO: This is the name if you download volar-server from the AUR
+-- need to wait for https://github.com/johnsoncodehk/volar/issues/458
+local bin_name = 'volar-server'
+
+-- TODO: Figure out how to load this dynamically
+local typescript_server_path = '/home/artur/.npm-global/lib/node_modules/typescript/lib/tsserverlibrary.js'
+
+local volar_init_options = {
+  typescript = {
+     serverPath = typescript_server_path,
+   },
+   languageFeatures={
+     references = true,
+     definition = true,
+     typeDefinition = true,
+     callHierarchy = true,
+     hover = true,
+     rename = true,
+     renameFileRefactoring = true,
+     signatureHelp = true,
+     codeAction = true,
+     completion={
+       defaultTagNameCase='both',
+       defaultAttrNameCase='kebabCase',
+     },
+     schemaRequestService  = true ,
+     documentHighlight = true,
+     documentLink = true ,
+     codeLens = true,
+     semanticTokens = true,
+     diagnostics = true,
+   },
+   documentFeatures={
+     selectionRange = true,
+     foldingRange = true,
+     linkedEditingRange = true,
+     documentSymbol = true,
+     documentColor = true,
+     documentFormatting = {
+       defaultPrintWidth = 100,
+     },
+   }
+ }
+
+local volar_settings = {
+   volar= {
+     codeLens= {
+       references = true,
+       pugTools = true,
+       scriptSetupTools = true,
+     },
+     lowPowerMode = false,
+     tsPluginStatus = false,
+     checkVueTscVersion = false,
+     preview= {
+       port = 3333,
+       backgroundColor = "fff",
+       transparentGrid = false,
+     }
+   }
+ }
+
+configs[server_name] = {
+  default_config = {
+    cmd = { bin_name, '--stdio' },
+    filetypes = { 'vue' },
+    root_dir = util.root_pattern('package.json'),
+    init_options = volar_init_options,
+    settings = volar_settings,
+  },
+  docs = {
+    package_json = 'https://github.com/johnsoncodehk/volar/master/packages/server/package.json',
+    description = --TODO
+    [[
+      https://github.com/johnsoncodehk/volar/tree/master/packages/server
+      Volar language server for Vue
+      Volar can be installed via NPM
+      ```sh
+      npm install -g @volar/server
+      ```
```

Comment:
Improve docs

1. Mention that you can't run vuels and volarvuels at the same time
2. If we decide to not wait for Volar issue 453 (add executable) we need to mention that there's extra configuration needed (add your own volar-server .sh wrapper to PATH)